qiskit/pulse/__init__.py

```diff
@@ -108,17 +125,17 @@
 
    InstructionScheduleMap
 
-Rescheduling Utilities
+Pulse Program Transformations
 ======================
 
 These utilities return modified :class:`~qiskit.pulse.Schedule` s.
 
 .. autosummary::
    :toctree: ../stubs/
 
-   ~reschedule.align_measures
-   ~reschedule.add_implicit_acquires
-   ~reschedule.pad
+   ~transforms.align_measures
```

Comment:
This should probably go into the same issue with `scheduler` name to discuss with the team